packages/stream_video/lib/src/call/call_type.dart

```diff
@@ -0,0 +1,34 @@
+/// Default implementation of [CallType].
+///
+/// This provides the default call types used by the SDK.
+/// - [StreamCallType.Default()] (default)
+/// - [StreamCallType.liveStream] (livestream)
+/// - [StreamCallType.development] (development)
+/// - [StreamCallType.audioRoom] (audio_room)
+class StreamCallType {
+  const StreamCallType._(this.value);
+
+  final String value;
+
+  factory StreamCallType.Default() => const StreamCallType._('default');
```

Comment:
I know that we can't user 'default' starting with the lowercase letter but maybe we could use `defaultCall`? It breaks the pattern where factory constructor corresponds 1-1 to the string name of the type but Uppercase letter breaks the dart convention and looks bad.

Reply:
It is too verbose ... how about `StreamCallType.default_()`?

Reply:
Or just:
```dart
  factory StreamCallType() => const StreamCallType._('default');
 ```